### ISSUE TITLE
Mapping between Imagnet and AudioSet

### ISSUE THREAD

**Issue by XinyuLyu** [shvdiwnkozbw/Multi-Source-Sound-Localization#6]
Hi,
When I look into the  cluster_v3.npy, I found the mapping between 1000 categorys from Imagnet and 7 categorys of AudioSet is not consistent. For example, catefory banjo in Imagnet(ILSVRC2017) is maps to the third category(animal) in AudioSet. Can you help me with that?

**Comment by shvdiwnkozbw** [comment]
Yes, since the mapping is based on the similarity of word embeddings, there could be much noise and wrong correspondence in the mapping.
